backend/wmg/data/snapshot.py

```diff
@@ -0,0 +1,112 @@
+import logging
+import os
+from dataclasses import dataclass
+from typing import Optional
+
+import pandas as pd
+import tiledb
+from pandas import DataFrame
+from tiledb import Array
+
+from backend.corpora.common.utils.s3_buckets import buckets
+from backend.wmg.config import WmgConfig
+from backend.wmg.data.tiledb import create_ctx
+
+CELL_TYPE_ORDERINGS_FILENAME = "cell_type_orderings.json"
+EXPRESSION_SUMMARY_CUBE_NAME = "expression_summary"
+CELL_COUNTS_CUBE_NAME = "cell_counts"
+
+logger = logging.getLogger("wmg")
+
+
+@dataclass
+class WmgSnapshot:
+    """
+    All of the data artifacts the WMG API depends upon to perform its functions, versioned by "snapshot_identifier".
```

Comment:
I think it would be helpful to list all of the data artifacts (working name and short summary) here

Reply:
ok if I comment the constants above? those are the actual data artifact dir/file names.

Reply:
yea that would be great

Reply:
added comments above and below